### PR TITLE
EIP-1576: Simple Whisper bid negotiation protocol

### PR DESCRIPTION
**Pull request by djvs**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-X.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your Github username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-1576.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    }
+
+## Implementation
+
+Both protocols are implemented (with some negligible schema changes) as the Bloom Protocol [Attestation Kit](https://github.com/hellobloom/attestation-kit).  The Whisper-specific implementation code can be found [here](https://github.com/hellobloom/attestation-kit/tree/master/app/shared/whisper), with Typescript specifications for message types found [here](https://github.com/hellobloom/attestation-kit/blob/master/app/shared/whisper/msgTypes.ts), and the majority of the sample code [here](https://github.com/hellobloom/attestation-kit/blob/master/app/shared/whisper/requesterActions.ts) and [here](https://github.com/hellobloom/attestation-kit/blob/master/app/shared/whisper/attesterActions.ts).
+
+
+## Copyright
+
+Copyright and related rights waived via CC0.
```
Comment (nicksavers) [inline EIPS/eip-1576.md]: You may want to add a link to the content of CC0
https://creativecommons.org/publicdomain/zero/1.0/

File: EIPS/eip-1576.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+## Rationale
+
+A simple, lightweight and fully flexible protocol for messaging chaining, and then additionally for economic negotiations, was desired.  An original protocol of fixed-price-for-good negotiations was replaced with an abstract "bid_type" key and secondary parameters to accomodate any type of economic transaction.  Top-level fields in the protocol were reserved for key protocol features, while the "data" key hosts type-specific information.
+
+# Message Protocol
```
Comment (axic) [inline EIPS/eip-1576.md]: These seem to have the wrong size. I think both of these headings should be under `Specification`.

**Comment by axic** [comment]
@gluk256 @gballet can you review this?

**Comment by djvs** [comment]
Whisper functionality appears to have been removed from Geth?  Otherwise I'd be happy to move this forward.

**Pull request closed without merging**